src/utils/getCalendarMonthWeeks.js

```diff
@@ -1,6 +1,7 @@
 export default function getCalendarMonthWeeks(month, enableOutsideDays) {
-  const firstOfMonth = month.clone().startOf('month');
-  const lastOfMonth = month.clone().endOf('month');
+  const baseDate = month.clone().utcOffset(month.utcOffset());
```

Comment:
Sure! Do you think "`// set utc offset to get correct dates in future (when timezone changes)`" is enough (didn't want to do a really long comment)?